pom.xml

```diff
@@ -501,23 +504,44 @@
         </plugins>
       </build>
 
+
+      <dependencyManagement>
+        <dependencies>
+          <dependency>
+            <groupId>com.google.guava</groupId>
+            <artifactId>guava</artifactId>
+            <version>31.1-jre</version>
+          </dependency>
+        </dependencies>
+      </dependencyManagement>
+
       <dependencies>
+        <dependency>
+          <groupId>com.google.guava</groupId>
+          <artifactId>guava</artifactId>
+        </dependency>
+        <dependency>
+          <groupId>org.slf4j</groupId>
+          <artifactId>slf4j-api</artifactId>
+          <version>2.0.7</version>
```

Comment:
scope test missing

Reply:
Getting compilation error in Build when scope test is added, Verified in local. Hence removed the tag.